### PR TITLE
WFLY-10612 Include EJB's IIOP Binding when EJB is deployed logging

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-10612

### CODE REVIEW

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 1]
```diff
@@ -121,6 +121,12 @@
      */
     EjbLogger DEPLOYMENT_LOGGER = Logger.getMessageLogger(EjbLogger.class, "org.jboss.as.ejb3.deployment");
 
+    /**
+     * A logger with the category {@code org.jboss.as.ejb3.iiop} used for deployment
+     * log messages
+     */
+    EjbLogger IIOP_DEPLOYMENT_LOGGER = Logger.getMessageLogger(EjbLogger.class, "org.jboss.as.ejb3.iiop");
```
Comment (bstansberry) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:128]: This should just use DEPLOYMENT_LOGGER.

Comment (RanabirChakraborty) [reply ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java]: Thanks, Brian. I have changed accordingly.

**Comment by bstansberry** [comment]
@chengfang or @tadamski Please review.

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 2]
```diff
@@ -3204,4 +3204,8 @@
     @LogMessage(level = WARN)
     @Message(id = 515, value = "[EJB3.2 spec, section 4.9.2] Singleton session beans are not allowed to implement 'javax.ejb.SessionBean' interface. This interface on bean '%s' is going to be ignored and should be removed.")
     void singletonCantImplementSessionBean(String className);
+
+    @LogMessage(level = INFO)
+    @Message(id = 516, value = "IIOP bindings for session bean named '%s' in deployment unit '%s' are as follows: %s")
+    void iiopBindings(final String ComponentName, final String ModuleName, String Name);
```
Comment (chengfang) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:3210]: Just some minor style issues: either use final for all 3 params or no final.
`ComponentName` -> `componentName`

**Review by chengfang — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @RanabirChakraborty !

**Pull request merged**